pkgs/os-specific/linux/aic8800/default.nix

```diff
@@ -0,0 +1,49 @@
+{
+  lib,
+  stdenv,
+  kernel,
+  kernelModuleMakeFlags,
+  dpkg,
+  aic8800-firmware,
+}:
+stdenv.mkDerivation (finalAttr: {
+  name = "aic8800";
+  version = aic8800-firmware.version;
+  src = aic8800-firmware.src;
+
+  hardeningDisable = [ "pic" ];
+
+  nativeBuildInputs = kernel.moduleBuildDependencies ++ [ dpkg ];
+
+  makeFlags = kernelModuleMakeFlags ++ [
+    "KDIR=${kernel.dev}/lib/modules/${kernel.modDirVersion}/build"
+  ];
+
+  patchPhase = ''
+    runHook prePatch
+
+    dpkg-source --before-build .
+    cp -fv ${./Makefile} ./Makefile
+
+    runHook postPatch
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    find . -name "*.ko" -exec install {} -Dm444 -v -t $out/lib/modules/${kernel.modDirVersion}/kernel/drivers/aic8800 \;
```

Comment:
USB and SDIO has conflicting modules with same name. We renamed them with [`dkms.conf`](https://github.com/radxa-pkg/aic8800/blob/main/debian/aic8800-sdio-dkms.dkms#L18), but this looks like one will be overwritten by another.